negotiation/message/schema/contract-negotiation-error-schema.json

```diff
@@ -14,37 +14,40 @@
 			"properties": {
 				"@context": {
 					"type": "string",
-					"const": "https://w3id.org/dspace/v0.8/context.json",
-					"required": true
+					"const": "https://w3id.org/dspace/v0.8/context.json"
 				},
 				"@type": {
 					"type": "string",
-					"const": "dspace:ContractNegotiationError",
-					"required": true
+					"const": "dspace:ContractNegotiationError"
 				},
 				"dspace:providerPid": {
-					"type": "string",
-					"required": true
+					"type": "string"
 				},
 				"dspace:consumerPid": {
-					"type": "string",
-					"required": true
+					"type": "string"
+				},
+				"dspace:code": {
+					"type": "string"
 				},
 				"dspace:reason": {
 					"type": "array",
 					"minItems": 1,
-					"items": {
-
-					},
-					"required": false
+					"items": {}
 				},
 				"dct:description": {
 					"type": "array",
 					"items": {
-						"type": "string"
-					}
+						"@language": {
+							"type": "string"
+						},
+						"@value": {
+							"type": "string"
+						}
+					},
+					"required": [ "@language", "@value"]
 				}
-			}
+			},
+			"required": [ "@context", "@type", "dspace:providerPid", "dspace:consumerPid", "dspace:reason"]
```

Comment:
`dspace:reason` is not required (" The message can be sent at any state of a negotiation without providing an explanation.", [contract.negotiation.protocol.md](https://github.com/International-Data-Spaces-Association/ids-specification/blob/main/negotiation/contract.negotiation.protocol.md)). Same goes for the transfer error message.